css/classify.styl

```diff
@@ -124,6 +124,10 @@
   & + .workflow-task
     margin-top: 2em
 
+  .markdown:first-child
```

Comment:
Maybe use `.question:first-child` instead of `.markdown` so it has a more targeted scope? Not sure if markdown comes in somewhere else in the workflow task where a top margin is desirable.